app/dashboard/page.tsx

```
@@ -1,656 +1,622 @@
-                  <CardHeader>
-                    <CardTitle>Your AI Receptionist</CardTitle>
-                    <CardDescription>
-                      Customize how your AI assistant interacts with callers.
-                    </CardDescription>
-                  </CardHeader>
-                  <CardContent>
-                    {settingsLoading || !dataLoaded ? (
-                      <div className="flex items-center justify-center py-8">
+
+          {/* Analytics Section */}
+          {analytics && analytics.metrics && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)